### PR TITLE
DOC:  Document ria url formats in docstring

### LINKED ISSUE #6860: DOC: create-sibling-ria lacks examples
Related to #6800, I noticed that the typical ``examples`` that commands have is lacking in ``create-sibling-ria``. We should add a few, maybe even copy a few from [handbook.datalad.org/en/latest/r.html?RIA](http://handbook.datalad.org/en/latest/beyond_basics/101-147-riastores.html#riastore).

### PR DESCRIPTION
**Pull request by adswa**
@bpoldrack, I made a start at fixing #6800, which asks for docs for RIA URLs. Can you fill in the TODO in this docstring?

This PR can also be a place for additional commits that fix #6860 

#### 📝 Documentation
- create-sibling-ria's docstring now defines the schema of RIA URLs and clarifies internal layout of a RIA store.

### CODE REVIEW

**Comment by mih** [comment]
Ping @bpoldrack

File: datalad/distributed/create_sibling_ria.py [hunk 1]
```diff
@@ -84,6 +84,26 @@ class CreateSiblingRia(Interface):
...
+    'ria+[scheme]://<storelocation>' (e.g.,
+    ria+ssh://[user@]hostname:/absolute/path/to/ria-store, or
+    ria+file:///absolute/path/to/ria-store)
+
+    The general structure of a RIA URL pointing to a dataset in a store (for
+    example for cloning) takes a similar form, but appends either the datasets
+    UUID or a ~ symbol followed by the dataset's alias name:
+    'ria+[scheme]://<storelocation>#<dataset-UUID>' or
+    'ria+[scheme]://<storelocation>#~<aliasname>'.
+    In addition, specific version identifiers can be appended to the URL with an
```
Comment (adswa) [inline datalad/distributed/create_sibling_ria.py:103]: Citing the original issue:

>  Adding @something to a working URL seems to look for a branch with that name, seems not to work for other tree-ish (commit, tag).

Does it only work for branches? If if only works for branches, should that be made clear?

**Comment by adswa** [comment]
Thanks! I just re-raised one question from the original issue. And maybe we should also add examples. I can see if I have time today

File: datalad/distributed/create_sibling_ria.py (hunk 1 above)
Comment (bpoldrack) [reply datalad/distributed/create_sibling_ria.py:103]: True. It's passed onto the `--branch` option to `git clone`, which considers branches and tags.

File: datalad/distributed/create_sibling_ria.py [hunk 2]
```diff
@@ -84,6 +84,26 @@ class CreateSiblingRia(Interface):
...
+    additional @ symbol:
+    'ria+[scheme]://<storelocation>#<dataset-UUID>@<dataset-version>'
```
Comment (bpoldrack) [inline datalad/distributed/create_sibling_ria.py:105]:
```suggestion
    'ria+[scheme]://<storelocation>#<dataset-UUID>@<dataset-version>', where 'dataset-version' refers
    to a branch or tag.
```

File: datalad/distributed/create_sibling_ria.py (hunk 1 above)
Comment (adswa) [reply datalad/distributed/create_sibling_ria.py:103]: Would you mind pushing a clarification?

**Comment by mih** [comment]
I am putting this into draft mode. There has been no activity in a while. Please close, if abandoned.

**Comment by adswa** [comment]
I have pushed a commit rephrasing the layout section into something that I feel is more pragmatic/non-technical and less vaguely scary. It would be nice to get this vetted by @bpoldrack so that the tiny doc fix for #6800 can finally make it in.

**Comment by bpoldrack** [comment]
>  It would be nice to get this vetted by @bpoldrack so that the tiny doc fix for #6800 can finally make it in.

Yes, looks good to me. Thank you, @adswa.

**Comment by yarikoptic-gitmate** [comment]
PR released in [`0.18.0`](https://github.com/datalad/datalad/releases/tag/0.18.0)

**Pull request merged**